ipfsconn/ipfshttp/ipfshttp_test.go

```diff
@@ -291,13 +291,12 @@ func TestResolve(t *testing.T) {
 	ipfs, mock := testIPFSConnector(t)
 	defer mock.Close()
 	defer ipfs.Shutdown()
-	time.Sleep(2 * time.Second)
+	time.Sleep(time.Second)
```

Comment:
Sleep is not necessary. There is an error log, but it's unrelated (the test passes).